variables.tf

```diff
@@ -688,3 +688,15 @@ variable "ephemeral_storage_size" {
     error_message = "The minimum supported value is 21 GiB and the maximum supported value is 200 GiB."
   }
 }
+
+variable "alb_ip_address_type" {
+  description = "The type of IP addresses used by the subnets for your load balancer"
```

Comment:
Add available options (`ipv4` or `dualstack` - for IPv6) in the description.

Reply:
Done